src/main/java/net/sf/jabref/importer/fetcher/IEEEXploreFetcher.java

```diff
@@ -53,65 +56,32 @@
 public class IEEEXploreFetcher implements EntryFetcher {
 
     private static final Log LOGGER = LogFactory.getLog(IEEEXploreFetcher.class);
+    private static final String URL_SEARCH = "http://ieeexplore.ieee.org/rest/search?reload=true";
+    private static final String URL_BIBTEX_START = "http://ieeexplore.ieee.org/xpl/downloadCitations?reload=true&recordIds=";
+    private static final String URL_BIBTEX_END = "&download-format=download-bibtex&x=0&y=0";
+    private static final String DIALOG_TITLE = Localization.lang("Search %0", "IEEEXplore");
+    private static final int MAX_FETCH = 100;
 
     final CaseKeeperList caseKeeperList = new CaseKeeperList();
     private final CaseKeeper caseKeeper = new CaseKeeper();
     private final UnitFormatter unitFormatter = new UnitFormatter();
-
     private final HTMLConverter htmlConverter = new HTMLConverter();
-
     private final JCheckBox absCheckBox = new JCheckBox(Localization.lang("Include abstracts"), false);
 
-    private static final int MAX_FETCH = 100;
-    private final int perPage = IEEEXploreFetcher.MAX_FETCH;
     private int hits;
-    private int unparseable;
     private int parsed;
-    private int piv;
     private boolean shouldContinue;
     private boolean includeAbstract;
 
     private String terms;
-    private final String endUrl = "&rowsPerPage=" + Integer.toString(perPage) + "&pageNumber=";
-    private String searchUrl;
-
-    private final Pattern hitsPattern = Pattern.compile("([0-9,]+) Results");
-    private final Pattern typePattern = Pattern.compile("<span class=\"type\">\\s*(.+)");
-    private final HashMap<String, String> fieldPatterns = new HashMap<>();
-    private final Pattern absPattern = Pattern.compile("<p>\\s*(.+)");
-
-    Pattern stdEntryPattern = Pattern.compile(".*<strong>(.+)</strong><br>" + "\\s+(.+)");
+    private String postData;
```

Comment:
Can you try to convert the field variables as local variables in the method? With a quick check, I realized that this works for at least two fields.